### PR TITLE
RFC: Single-Page-Application HTTP/WebSocket API

### PR DESCRIPTION
**Pull request by luca-moser**

### CODE REVIEW

File: text/0000-spa-http-api/oas_http_rest_api_spec.yml [hunk 1]
```diff
@@ -0,0 +1,626 @@
...
+      description: Access token missing or invalid.
+  schemas:
+    NodeInfo:
+      type: object
+      properties:
+        app_name:
+          type: string
+          example: "HORNET"
+        app_version:
+          description: The node's semver version.
```
Comment (kwek20) [inline text/0000-spa-http-api/oas_http_rest_api_spec.yml]: server

Comment (kwek20) [reply text/0000-spa-http-api/oas_http_rest_api_spec.yml]: or is semver something i dont know? hmm

Comment (luca-moser) [reply text/0000-spa-http-api/oas_http_rest_api_spec.yml]: https://semver.org/ :)

Comment (kwek20) [reply text/0000-spa-http-api/oas_http_rest_api_spec.yml]: aaaaaah i half thought so but :shrug:

**Pull request closed without merging**